### ISSUE TITLE
vweb reduces a received file size from http post. (The file is corrupted.)

### ISSUE THREAD

**Issue by kahsa** [vlang/v#12423]
**V version:0.2.4 6c244d3** 
**OS:ArchLinux** 

**What did you do?**
```html
<!DOCTYPE html>

<head>
	<meta charset="utf-8">
	<!-- <script type="text/javascript" src="/js_css/new.js"></script> -->
	<script>
		function send() {
			var fd = new FormData(document.querySelector('#item_form'));
			pic = document.querySelector('#item_picture').files[0];
			fd.append('picture_size', pic.size);
			var xhr = new XMLHttpRequest();
			xhr.open("post", "/new_item_test2");
			xhr.send(fd);
		}
	</script>
</head>

<body>
	<form id="item_form">
		<input type="file" name="picture" accept="image/jpeg" id="item_picture"><br>
	</form>
	<button onclick="send()">send</button><br>
</body>

</html>
```

```v
[post]
pub fn (mut app App) new_item_test2(picture_size int) vweb.Result {
	picture := app.files['picture'][0]
	println('original picture size: $picture_size')
	println('${picture.filename}: ${picture.data.len}')
	println('diff: ${picture.data.len - picture_size}')
	os.write_file(picture.filename, picture.data) or { panic(err)}
	return app.redirect('/dashboard/items')
}
```

**What did you expect to see?**

same size.
 
**What did you see instead?**
```sh
original picture size: 2458748
20181010_134404.jpg: 2458737
diff: -11
```

**Comment by kahsa** [comment]
maybe related to this, https://github.com/vlang/v/tree/master/examples/vweb/file_upload is fall in infinite loop reading 3MB jpeg file.

**Comment by kahsa** [comment]
https://discord.com/channels/592103645835821068/697813437237166131/907969998121738262

**Comment by Dialga** [comment]
Related #9934.

**Comment by kahsa** [comment]
fixed by 466ced2876ab1febbcc557d724069a4a9ed99337
